src/Fhir.Anonymizer.Core/AnonymizationConfigurations/AnonymizerRule.cs

```diff
@@ -9,13 +9,16 @@ public class AnonymizerRule
         public AnonymizerRuleType Type { get; set; }
 
         public string Source { get; set; }
+        // Rule priority, based on line sequence
```

Comment:
Will we have separate path and type rules even with priority defined by line sequence?

Reply:
For now, we prefer to keep seperately as paths in path rules are compatiable with FHIR paths and the type rules are not.  And we might need to implement wildcard and conditional anonymization taking advantage of FHIRPath compiler which is powerful with functions, so we'd better leave them seperately for future compatibility as there will be incremental changes to the configuration.
We will merge these rules finally when the configuration file comes to a steady state.